python/pacemaker/_cts/CTS.py

```diff
@@ -194,22 +201,23 @@ def wait_for_all_nodes(self, nodes, timeout=300):
 
 
 class Process:
-    """ A class for managing a Pacemaker daemon """
+    """A class for managing a Pacemaker daemon"""
 
     # pylint: disable=invalid-name
-    def __init__(self, cm, name, dc_only=False, pats=None, dc_pats=None,
-                 badnews_ignore=None):
-        """ Create a new Process instance.
-
-            Arguments:
-
-            cm              -- A ClusterManager instance
-            name            -- The command being run
-            dc_only         -- Should this daemon be killed only on the DC?
-            pats            -- Regexes we expect to find in log files
-            dc_pats         -- Additional DC-specific regexes we expect to find
-                               in log files
-            badnews_ignore  -- Regexes for lines in the log that can be ignored
+    def __init__(
+        self, cm, name, dc_only=False, pats=None, dc_pats=None, badnews_ignore=None
+    ):
```

Comment:
I don't like it either but that's how black does things. You should see the pcs code since they started using black formatting.

I value consistency in style above any particular preferences, so I'm fine with this if we stick with black.

Reply:
It does avoid the long function name problem. I think it looks fine. I'd rather stay consistent with our C style to avoid having to remember differently here, but it's not a big deal since we can easily run black on new code.